zh/deploy-on-aws-eks.md

```diff
@@ -374,9 +393,9 @@ parameters:
   fsType: ext4
 ```
 
-然后在 tidb cluster 的 YAML 文件中，通过 `storageClassName` 字段指定 `io1` 存储类申请 `io1` 类型的 EBS 存储。
+然后在 tidb cluster 的 YAML 文件中，通过 `storageClassName` 字段指定 `io1` 存储类申请 `io1` 类型的 EBS 存储。可以参考 [Tiflash 部署文档](deploy-tiflash.md)。
```

Comment:
It's kind of awkward to point users to the TiFlash deployment doc (seemingly irrelevant) for configuring `storageClassName`.

Do you think we can directly put the configuration example here?